content/guides/10-minute-tutorial.md

```diff
@@ -104,6 +118,35 @@ Open the project in IntelliJ IDEA:
 
 {{% /block %}}
 
+{{% block "kotlin" %}}
+
+To use Kotlin in our project, we need to take some extra steps:
+
+* Add a directory named `kotlin` in your `src/test` directory and mark it as `Test Sources Root`.
+In IntelliJ, you can do so by right-clicking on the `kotlin` directory and selecting **"Mark Directory as" > "Test Sources Root"**.
+* Create the `hellocucumber` package inside the `kotlin` directory.
+* Create a Kotlin class called `RunCucumberTest` inside the `hellocucumber` package and copy the annotations from the `RunCucumberTest.java` class to the `RunCucumberTest.kt` class.
```

Comment:
Is this list complete? I believe that some more setup is required to actually get the maven project to work. I guess it works directly with IDEA but that would leave things incomplete.

From the kotlin test module:

https://github.com/cucumber/cucumber-jvm/blob/master/kotlin-java8/pom.xml#L41

Reply:
Thanks. I missed a step where IntelliJ warns that Kotlin is not configured yet and adds Kotlin to the pom.xml when you click "Configure" on that warning. 
I've added that step and the resulting `pom.xml`.